lib/views/after_auth_screens/chat/chat_list_screen.dart

```diff
@@ -1,20 +1,15 @@
-// ignore_for_file: talawa_api_doc
-// ignore_for_file: talawa_good_doc_comments
-
 import 'package:flutter/material.dart';
 import 'package:talawa/views/after_auth_screens/chat/direct_chats.dart';
-import 'package:talawa/views/after_auth_screens/chat/event_chats.dart';
 import 'package:talawa/views/after_auth_screens/chat/select_contact.dart';
 
-/// ChatPage returns a stateless widget for current user Chat Page which renders
-/// the list of all the users that the current user has chat with.
+/// Renders the list of users with whom the current user has chatted.
```

Comment:
Please remove "renders the". It's a widget, not a rendering function. Yes the widget will be rendered, but all of the UI is rendered, obviously.

Reply:
Please check now!